extensions/wikia/AdEngine/src/templates/big-fancy-ad-below-mobile-config.js

```diff
@@ -0,0 +1,32 @@
+import { context } from '@wikia/ad-engine';
+
+function getNavbarHeight() {
+	const navbar = document.querySelector('.site-head-wrapper');
+
+	if (navbar) {
+		return navbar.offsetHeight;
+	}
+
+	return 0;
+}
+
+function getUnstickThreshold() {
+	const viewportWidth = Math.max(document.documentElement.clientWidth, window.innerWidth, 0);
+
+	return (viewportWidth * 9 / 16 + getNavbarHeight()) * 2;
```

Comment:
Out of curiosity, where is the 9/16 coming from?

Reply:
It's an aspect ratio of BFAA default state on mobile